src/pages/event/[id].tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Loader } from "../../components/Loader";
+import { useEffect, useState } from "react";
+import { useRouter } from "next/router";
+import type { Event } from "../../model/Event";
+
+const EventDetail: React.FC = () => {
+  const router = useRouter();
+  const { id } = router.query;
+  const [event, setEvent] = useState<Event>({} as Event);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchEvent = async () => {
+      const response = await fetch(
+        `https://sportujspolu-api.onrender.com/api/v1/events/${id}`
```

Comment:
For now. We have two fetches. This is fine. With the third one I would create an API layer. That means a folder with all the routes separated by the namespace

So `/v1/events` would go to `/api/events.ts` file

Keep this for now

Reply:
understood